README.md

```diff
@@ -68,17 +68,19 @@ This repo also includes related tools for [running tasks](#kubernetes-run) and [
 ## Prerequisites
 
 * Ruby 2.3+
-* Your cluster must be running Kubernetes v1.6.0 or higher
+* Your cluster must be running Kubernetes v1.7.0 or higher<sup>1</sup>
```

Comment:
I had been doing it on master right before releasing, but it would indeed make particular sense to add it up-front in the PR in this case, since this PR adds documentation that mentions version 0.16.